templates/inactivity_body.mjml

```diff
@@ -0,0 +1,62 @@
+<mjml>
+  <mj-head>
+    <mj-include path="./partials/style.mjml" />
+  </mj-head>
+  <mj-body css-class="emailBody">
+    <mj-wrapper mj-class="email">
+      <mj-include path="./partials/logo.mjml" />
+
+      <mj-section padding="0px 24px 40px 24px">
+        <mj-column>
+          <mj-text css-class="title" align="left" color="#3F4350" font-size="16px" font-weight="600" line-height="24px" padding-bottom="16px">
+            {{.Props.Title}}
+          </mj-text>
+          <mj-text align="left" color="#3F4350" font-size="16px" font-weight="normal" line-height="24px" padding-bottom="16px">
+            {{.Props.SubTitle}}
+          </mj-text>
+          <mj-text align="left" color="#3F4350" font-size="16px" font-weight="normal" line-height="24px" padding-bottom="16px">
+            <ul>
+              <li>{{.Props.InfoBullet}}<b>Channels</b></li>
+              <li>{{.Props.InfoBullet1}}<b>Playbooks</b></li>
+              <li>{{.Props.InfoBullet2}}<b>Boards</b></li>
```

Comment:
These might not translate well since the subject (Channels, Playbooks, Boards) could move around in different languages.

Reply:
Since these are product names, I am wondering if they should be translated. @cwarnermm thoughts?

Reply:
@AGMETEOR - Great catch and thanks for asking! Official product names, including Channels, Playbooks, and Boards aren't translated into other languages.

Reply:
@cwarnermm to clarify, while the product names don't get translated, they should be translated as part of the entire string, right? 

For example:
"Guest access to specified **Channels**"
Could be something like:
"accès invité aux **Channels** publiques" in french - where "Channels" is no longer at the end of the sentence for it to make sense?

Reply:
@AGMETEOR this hasn't been addressed. 

You either need to: 
Include the subject (ie, Channels, Playbooks, Boards, etc directly into the translation ["Guest access to specified Channels"](https://github.com/mattermost/mattermost-server/pull/19374/files#diff-1a490f08504158838befaa816ee8d5c08dff9e694d5a568ce4b0a4398dcc563eR3572) You can [bold like this](https://github.com/mattermost/mattermost-server/blob/daa19220cce18409f3f2aff6086d6d4e45ba4fd5/i18n/en.json#L2560)

or

Pass the subject in as a variable ["Guest access to specified {item}"](https://github.com/mattermost/mattermost-server/pull/19374/files#diff-1a490f08504158838befaa816ee8d5c08dff9e694d5a568ce4b0a4398dcc563eR3572) and then passing in "Channel" like you do with `Name` [here](https://github.com/mattermost/mattermost-server/pull/19374/files#diff-9eea96507e6338e4e5e7e0bb9a00794fcf4062ef1b710c68fc300377823f4233R771)